systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2IsolatedST.java

```diff
@@ -1129,6 +1121,28 @@ void testKMM2RollAfterSecretsCertsUpdateScramSha(ExtensionContext extensionConte
         LOGGER.info("Messages successfully mirrored");
     }
 
+    private static void modifyKafkaUserPasswordWithNewSecret(String namespace, String kafkaUserSourceName, String customSecretSource, String customPassword) {
```

Comment:
In the test method you are using once `kafkaUserSourceName` and then `kafkaUserTargetName`
I would change the `kafkaUserSourceName` to something more generic
Same for the `customSecretSource`